test/e2e/broker_default_test.go

```diff
@@ -73,16 +73,16 @@ func TestDefaultBrokerWithManyTriggers(t *testing.T) {
 	// These are the event types and sources that triggers will listen to, as well as the selectors
 	// to set  in the subscriber and services pods.
 	eventsToReceive := []eventReceiver{
-		{base.TypeAndSource{Type: any, Source: any}, newSelector()},
-		{base.TypeAndSource{Type: eventType1, Source: any}, newSelector()},
-		{base.TypeAndSource{Type: any, Source: eventSource1}, newSelector()},
-		{base.TypeAndSource{Type: eventType1, Source: eventSource1}, newSelector()},
+		{resources.TypeAndSource{Type: any, Source: any}, newSelector()},
```

Comment:
same with this.. but as you wish

Reply:
Moved `TypeAndSource` to the `broker_default_test.go` since it's only used by this test case.